rest/src/main/java/com/redhat/ipaas/rest/v1/handler/BaseHandler.java

```diff
@@ -25,6 +25,13 @@
     @Autowired
     private DataManager dataMgr;
 
+    public BaseHandler() {
+    }
+
+    public BaseHandler(DataManager dataMgr) {
```

Comment:
Yeah I much prefer constructor injection where reference doesn't change during lifetime. Let me change that.